### PR TITLE
docs: Add guide to showcase handling outputs

### PR DESCRIPTION
**Pull request by marcosnils**
Add a guide which highlights how to handle action's outputs by writing
them to the client filesystem and using CUE's integrations to marshal
them as JSON.

Signed-off-by: Marcos Lilljedahl <marcosnils@gmail.com>

cc @helderco @jpadams

### CODE REVIEW

File: docs/tests/core-concepts/client/plans/output_marshal.cue.fragment [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#SomeAction: {
```
Comment (TomChv) [inline docs/tests/core-concepts/client/plans/output_marshal.cue.fragment:1]: That was a good idea to simplify example!

**Review by TomChv — APPROVED** [review]
LGTM

File: docs/tests/core-concepts/client/plans/output_marshal.cue.fragment (hunk 1 above)
Comment (marcosnils) [reply docs/tests/core-concepts/client/plans/output_marshal.cue.fragment:1]: Following @shykes flow here :D

**Review by shykes — APPROVED** [review]
Looks great! Article ID 1226 was taken, so I pushed an extra commit to use another one.

**Comment by shykes** [comment]
I also added your article to `website/sidebars.js`, otherwise it won't be visible on the navigation bar.

**Comment by shykes** [comment]
Oops, the sidebar name changed from "europa" to "0.2", had to change your markdown header accordingly.

This is a VIP PR review experience @marcosnils 😂 I'm too invested, I want to merge this article now!

**Comment by shykes** [comment]
Looks like article 1227 (unrelated to this PR) was also referencing the wrong sidebar, causing deploy preview to break 🤔 Fixing that too.

File: website/sidebars.js [hunk 2]
```diff
@@ -53,6 +53,8 @@ module.exports = {
         "guides/custom-buildkit",
         "guides/self-signed-certificates",
         "guides/pushing-plan-dependencies",
+        "guides/coding-style",
```
Comment (helderco) [inline website/sidebars.js:56]: This shouldn't be here.

File: docs/guides/1228-handling-outputs.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+displayed_sidebar: 0.2
+---
+
+# Handling action outputs
+
+It's most likely that upon executing a Dagger action, you'll find in the situation of having to parse some outputs. In Dagger 0.1 (pre-Europa), inputs and outputs had to be specifically defined; starting Dagger 0.2 (Europa) we introduced a brand new API which removed this capability temporarily until we come up with the [right UX](https://github.com/dagger/dagger/issues/1351) to bring this back. Until the API is ready, you can follow this guide which shows different **temporary** alternatives of parsing dagger outputs.
+
+## Writing outputs to the filesystem
+
+As showcased in the [interacting with the client](/1203/client) docs, Dagger already has the ability to write into the client filesystem through the `client` API. Using this capability we can then output any value into our local filesystem for further automation.
```
Comment (helderco) [inline docs/guides/1228-handling-outputs.md:12]: Should point to .md file.

**Pull request merged**